CleanCode.SolidPrinciples/SingleResponsibility/HolidayLaugh.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Linq;
+
+namespace CleanCode.SolidPrinciples.SingleResponsibility
+{
+    class HolidayLaugh
```

Comment:
Nie bardzo widzę czemu miała być powiązana z klasą Holiday ciezko to okreslić trzeba byloby sie autora poczatkowego kodu zapytać czy taki byl zamysł.